uninstaller.nsh

```diff
@@ -0,0 +1,7 @@
+!macro customUnInstall
+    RMDir /r "$APPDATA\remoteit"
+    RMDir /r "$INSTDIR"
+    RMDir /r "$PROFILE\AppData\Local\remoteit"
+    RMDir /r "$PROFILE\AppData\Local\remoteit-desktop-updater"
```

Comment:
Does `remoteit-desktop-updater` get removed by the electron installer? Desktop doesn't add this file.

Also I don't see the paths here for:
`C:\ProgramData\remoteit\` and `C:\Program Files\remoteit-bin\`?

Reply:
@mauricio0000 I just tested and `C:\ProgramData\remoteit\` WAS removed. But `C:\Program Files\remoteit-bin\` is still there.

Reply:
@mauricio0000 Also like we discussed in Slack please have it run this first:
`\Program Files\remoteit-bin\remoteit.exe -j uninstall --yes`